chef-provisioning-lxc.gemspec

```diff
@@ -10,11 +10,9 @@ Gem::Specification.new do |s|
   s.description = s.summary
   s.author = 'Ranjib Dey'
   s.email = 'dey.ranjib@gmail.com'
-  s.homepage = 'https://github.com/opscode/chef-provisioning-lxc'
+  s.homepage = 'https://github.com/chef/chef-provisioning-lxc'
 
-  s.add_dependency 'chef', '>= 12.0.3'
   s.add_dependency 'chef-provisioning'
```

Comment:
In all the other projects we have `~> 1.0` if they work with 1.0 - can you update that here?